udata/tests/frontend/test_user_frontend.py

```diff
@@ -64,6 +64,12 @@ def test_render_profile(self):
         self.assertEquals(json_ld['url'], 'http://www.datagouv.fr/user')
         self.assertEquals(json_ld['image'], 'http://www.datagouv.fr/avatar')
 
+    def test_render_profile_of_an_inactive_user(self):
```

Comment:
`test_cant_render` or `test_dont_render` ?
Might be great to also test that admins can still display it.